src/en/authors-charm-policy.md

```diff
@@ -37,11 +37,11 @@ Not following these guidelines will result in a WARNING (W:) in `charm test`:
 - Should be built using [charm layers](authors-charm-building.html).
 - Should be delivered using Juju Resources by default. 
 
-## Testing and Quality Requirements
+## Testing and Quality Guidelines
```

Comment:
In general, we try to keep capitalization to a minimum. Only the 'T' should be capitalized here. Review all section headers.